test/Dockerfile-test

```diff
@@ -0,0 +1,13 @@
+# use SCP base Rails image, configure only project-specific items here
+FROM gcr.io/broad-singlecellportal-staging/single-cell-portal:development
```

Comment:
FWIW, like the reference in the GitHub Actions .yml file, this case of using "staging" and "development" in the name / version for the Docker image doesn't alarm me.